### PR TITLE
feat: add tutorials section

### PR DESCRIPTION
**Pull request by Ashutosh-Bhadauriya**
Added tutorials section

### CODE REVIEW

File: .DS_Store
Comment (FlxMgdnz) [inline .DS_Store:1]: Should this file be part of the PR?

File: images/.DS_Store
Comment (FlxMgdnz) [inline images/.DS_Store:1]: Should this file be part of the PR?

File: .DS_Store
Comment (Ashutosh-Bhadauriya) [reply .DS_Store:1]: No this shouldn't be

File: images/.DS_Store
Comment (Ashutosh-Bhadauriya) [reply images/.DS_Store:1]: Just deleted those

**Review by FlxMgdnz — APPROVED** [review]

**Pull request merged**